src/current/v24.2/ui-overload-dashboard.md

```diff
@@ -15,93 +21,113 @@ To view this dashboard, [access the DB Console]({% link {{ page.version.version
 
 The **Overload** dashboard displays the following time series graphs:
 
-## CPU percent
+## CPU Utilization
 
 {% include {{ page.version.version }}/ui/cpu-percent-graph.md %}
 
-## Goroutine Scheduling Latency: 99th percentile
+## KV Admission CPU Slots Exhausted Duration Per Second
 
-This graph shows the 99th [percentile](https://wikipedia.org/wiki/Percentile#The_normal_distribution_and_percentiles) of scheduling latency for [Goroutines](https://golangbot.com/goroutines/) as tracked by the `cr.node.go.scheduler_latency-p99` metric.
+This graph shows the relative time the node had exhausted slots for foreground (regular) CPU work per second of wall time, measured in microseconds/second, as tracked by the `admission.granter.slots_exhausted_duration.kv` metric. Increased slot exhausted duration indicates CPU resource exhaustion.
 
-- In the node view, the graph shows the 99th percentile of scheduling latency for Goroutines on the selected node.
+KV admission slots are an internal aspect of the admission control system, and are dynamically adjusted to allow for high CPU utilization, but without causing CPU overload. If the used slots are often equal to the available slots, then the admission control system is queueing work in order to prevent overload. A shortage of KV slots will cause queuing not only at the KV layer, but also at the SQL layer, since both layers can be significant consumers of CPU.
 
-- In the cluster view, the graph shows the 99th percentile of scheduling latency for Goroutines across all nodes in the cluster.
+- In the node view, the graph shows the admission slots exhausted duration in microseconds/second on the selected node.
+- In the cluster view, the graph shows the admission slots exhausted duration in microseconds/second across all nodes in the cluster.
 
-## Runnable Goroutines per CPU
+## Admission IO Tokens Exhausted Duration Per Second
 
-{% include {{ page.version.version }}/ui/runnable-goroutines-graph.md %}
+This graph shows the relative time the node had exhausted IO tokens for all IO-bound work per second of wall time, measured in microseconds/second, as tracked by the `admission.granter.io_tokens_exhausted_duration.kv` and `admission.granter.elastic_io_tokens_exhausted_duration.kv` metrics. There are separate lines for regular IO exhausted duration and elastic IO exhausted duration. Increased IO token exhausted duration indicates IO resource exhaustion.
+
+This graph indicates write IO overload, which affects KV write operations to storage. The admission control system dynamically calculates write tokens (similar to a [token bucket](https://wikipedia.org/wiki/Token_bucket)) to allow for high write throughput without severely overloading each store. This graph displays the microseconds per second that there were no write tokens left for arriving write requests. When there are no write tokens, these write requests are queued.
```

Comment:
suggest making 'storage' a link to our storage layer docs: https://www.cockroachlabs.com/docs/stable/architecture/storage-layer